repositories/modifiers.ts

```diff
@@ -0,0 +1,15 @@
+import { Filter } from 'mongodb';
```

Comment:
would rename this to utils.

Reply:
IMO naming a file 'utils' isn't good practice - the name doesn't have much meaning